src/app/location/templates/Station.html

```diff
@@ -3,7 +3,7 @@ <h4 class='heading'>Station</h4>
     <p class="help-block">Select a station by clicking on the map above.</p>
     <div class="form-inline">
         <div class="form-group">
-            <input type='text' disabled data-dojo-attach-point='stationTxt' class='form-control'>
+            <span class="text-danger required">*</span><input type='text' disabled data-dojo-attach-point='stationTxt' class='form-control'>
```

Comment:
Looks like there are some alignment issues here:
![image](https://user-images.githubusercontent.com/1326248/36906527-5f80da5e-1df3-11e8-833e-e0e4f32b2a63.png)

Reply:
fixed this with css